wavefront/server/modules/plugins_module/plugins_module/services/configuration_service.py

```
@@ -0,0 +1,172 @@
+
+    async def get_value(self, namespace: str, key: str) -> Optional[Any]:
+        """Return the config document, or None if there is no such key.
+
+        This is the hot path — every workflow run that uses a config node hits it.
+        """
+        cache_key = get_configuration_cache_key(namespace, key)
+        try:
+            cached = self.cache_manager.get_str(cache_key)
+            if cached is not None:
+                return json.loads(cached)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)